cpp/include/Ice/Logger.h

```diff
@@ -27,22 +28,22 @@ namespace Ice
         // send the message to C APIs that require null-terminated strings.
         // The message itself is also often constructed from a string produced by an ostringstream.
 
-        /// Print a message. The message is printed literally, without any decorations such as executable name or time
+        /// Prints a message. The message is printed literally, without any decorations such as executable name or time
         /// stamp.
```

Comment:
Shouldn't timestamp be a single word here?

Reply:
yes